packages/rancher-alerting-drivers/charts/Chart.yaml

```diff
@@ -6,15 +6,17 @@ annotations:
 apiVersion: v2
 appVersion: 1.16.0
 dependencies:
-- name: prom2teams
+- condition: prom2teams.enabled
```

Comment:
condition is optional so we did not see any issue in the previous version of the chart.